### PR TITLE
Add cmake build configuration

### PR DESCRIPTION
**Pull request by p12tic**
cc @simogasp

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.11)
+
+project(CoinUtils LANGUAGES C CXX)
+
+set(PROJECT_VERSION 2.10.14)
+set(PROJECT_VERSION_MAJOR 2)
+set(PROJECT_VERSION_MINOR 10)
+set(PROJECT_VERSION_PATCH 14)
```
Comment (simogasp) [inline CMakeLists.txt]:
```suggestion
project(CoinUtils LANGUAGES C CXX VERSION 2.10.14)
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+include(CheckIncludeFileCXX)
+include(CheckIncludeFile)
+check_include_file_cxx("cmath"    HAVE_CMATH)
+check_include_file_cxx("cfloat"    HAVE_CFLOAT)
+if(NOT HAVE_CMATH OR NOT HAVE_CFLOAT)
+    message(FATAL_ERROR "Missing necessary include files, like cmath or cfloat")
+endif()
+
+file(GLOB UTILS_SOURCES
```
Comment (simogasp) [inline CMakeLists.txt]: same comment as the other repos, it's up to you if you want to do it

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+set(CMAKE_CXX_STANDARD 14)
+
+include(GNUInstallDirs)
+set(INSTALL_INCLUDE_DIR ${CMAKE_INSTALL_INCLUDEDIR}/Coin)
+set(INSTALL_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
+
+if(MSVC)
+  set(
+    CMAKE_CXX_FLAGS
+    "${CMAKE_CXX_FLAGS} /bigobj /MP /wd4996 /wd4819 /wd4251 /wd4267 /wd4244 /wd4275"
```
Comment (simogasp) [inline CMakeLists.txt]: nitpicking, I'd use `target_compile_options` and move this after the declaration of the target
```suggestion
  target_compile_options(${PROJECT_NAME} PRIVATE /bigobj /MP /wd4996 /wd4819 /wd4251 /wd4267 /wd4244 /wd4275)
```
and the same for the APPLE below.
But it's really if you are up to it, the current code does the job too

**Review by simogasp — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (p12tic) [reply CMakeLists.txt]: Fixed now.

File: CMakeLists.txt (hunk 2 above)
Comment (p12tic) [reply CMakeLists.txt]: Fixed now.

File: CMakeLists.txt (hunk 3 above)
Comment (p12tic) [reply CMakeLists.txt]: Fixed now.

**Comment by p12tic** [comment]
@simogasp I've addressed your comments.

**Review by simogasp — APPROVED** [review]
Thanks!

**Pull request merged**